src/Services/Basket/Basket.FunctionalTests/Basket.FunctionalTests.csproj

```diff
@@ -17,8 +17,14 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.Authorization" Version="2.2.0" />
```

Comment:
Please remove unneeded packages

Reply:
INHO, these are need to resolve version conflicts

Reply:
They are not.

These are the conflicts both with or without these references:

```txt
Basket.FunctionalTests.csproj : warning NU1608: Detected package version outside of dependency constraint: Microsoft.ApplicationInsights.PerfCounterCollector 2.5.1 requires Microsoft.ApplicationInsights (= 2.5.1) but version Microsoft.ApplicationInsights 2.6.1 was resolved.
WebMVC.csproj : warning NU1608: Detected package version outside of dependency constraint: Microsoft.ApplicationInsights.PerfCounterCollector 2.5.1 requires Microsoft.ApplicationInsights (= 2.5.1) but version Microsoft.ApplicationInsights 2.6.1 was resolved.
Basket.API.csproj : warning NU1608: Detected package version outside of dependency constraint: Microsoft.ApplicationInsights.PerfCounterCollector 2.5.1 requires Microsoft.ApplicationInsights (= 2.5.1) but version Microsoft.ApplicationInsights 2.6.1 was resolved.
Locations.API.csproj : warning NU1608: Detected package version outside of dependency constraint: Microsoft.ApplicationInsights.PerfCounterCollector 2.5.1 requires Microsoft.ApplicationInsights (= 2.5.1) but version Microsoft.ApplicationInsights 2.6.1 was resolved.
```

No real surprises here, since conflicts are related to AI, changes in non-AI-related packages should make no difference.